xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/XWikiLivedata.vue

```diff
@@ -61,6 +61,8 @@ export default {
 
   props: {
     logic: Object,
+    xObjectPropertyHelper: Object,
```

Comment:
`displayerXObjectPropertyHelper.js` is currently provided by the webjar because it needs 'jquery', 'xwiki-meta', and 'xwiki-entityReference', but afaik the only way we have to pass a webjar dependency to a Vue component is to add it to the `provide` attributes of the root component (XWikiLivedata).
I agree this does not look great but I haven't find an cleaner way to far.